server/seeds/demoSeed.js

```diff
@@ -0,0 +1,22 @@
+const User = require("../models/User");
+const mongoose = require("mongoose");
+const createDemoUserDetails = async () => {
+  await mongoose.connect(process.env.MONGO_URI);
+  const email = process.env.DEMO_USER_EMAIL;
+  const password = process.env.DEMO_USER_PASSWORD;
+  const name = "Demo Account";
+
+  await User.create({
+    name,
+    email,
+    password,
+  })
```

Comment:
Also need to create a profile too - the profile can have any details hard coded.